www-client/w3m/w3m-0.5.3_p20210102.ebuild

```diff
@@ -0,0 +1,128 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools prefix toolchain-funcs vcs-snapshot
```

Comment:
Are we actually using vcs-snapshot?

Reply:
It isn't necessary anymore

Reply:
It's required otherwise it will fails to find the package

Reply:
Fixed. I will upload as soon as I understand why eautoreconf doesn't work